Sources/MessagingInApp/Gist/Managers/MessageQueueManager.swift

```diff
@@ -15,8 +16,9 @@ protocol MessageQueueManager: AutoMockable {
 }
 
 // sourcery: InjectRegisterShared = "MessageQueueManager"
+// sourcery: InjectSingleton
```

Comment:
This class has always been used as a singleton. The whole SDK would use `Gist.messageQueue`. To make the code testable, I switched to instead make the class a singleton in the digraph as we do for all other code in the SDK.